test/nuxeo-tooltip-test.html

```diff
@@ -0,0 +1,77 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+<head>
+  <title>nuxeo-tooltip tests</title>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <script>
+    // force use of polyfill: see ELEMENTS-56
+    window.fetch = null;
+  </script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../nuxeo-elements/test/test-helpers.js"></script>
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../widgets/nuxeo-tooltip.html">
+</head>
+<body>
+
+  <test-fixture id="hidden">
+    <template>
+      <nuxeo-tooltip hidden="true">Hello</nuxeo-tooltip>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="notHidden">
+    <template>
+      <nuxeo-tooltip>Hello</nuxeo-tooltip>
+    </template>
+  </test-fixture>
+
+
+  <script>
+    /* eslint-disable no-unused-expressions */
+
+    suite('<nuxeo-tooltip>', () => {
+      let element;
+
+      test('Should not show a paper-tooltip if it has hidden atrribute set to true', () => {
+        element = fixture('hidden');
+
+        element.show();
+        Polymer.dom.flush(element);
+        const paperTooltip = document.body.querySelector('paper-tooltip');
+        expect(paperTooltip).to.be.null;
+      });
+
+      test('Should show a paper-tooltip with the correct content if no hidden attribute is set', () => {
+        element = fixture('notHidden');
+
+        element.show();
+        Polymer.dom.flush(element);
+        const paperTooltip = document.body.querySelector('paper-tooltip');
+
+        expect(paperTooltip).to.exist;
+        expect(paperTooltip.innerHTML).to.equal('Hello');
```

Comment:
It would be interesting to also check the visibility.
We don't have the helper method as on master, but it shouldn't be hard to check. (it may also worth the have it on this repo)

Reply:
added it.